boto/endpoints.json

```diff
@@ -410,8 +434,9 @@
     },
     "sts": {
         "ap-northeast-1": "sts.amazonaws.com",
-        "ap-southeast-1": "sts.amazonaws.com",
-        "ap-southeast-2": "sts.amazonaws.com",
+        "ap-northeast-1": "sts.ap-northeast-2.amazonaws.com",
```

Comment:
Let's stick with using the global endpoint here to be consistent with the rest of the regions.